projects/scion/microfrontend-platform.e2e/src/messaging/intent-based-messaging-specs.ts

```diff
@@ -154,7 +169,8 @@ export namespace IntendBasedMessagingSpecs {
   }
 
   /**
-   * Tests that an application can issue intents to its public capabilities.
+   * Tests that an application can issue intents to its public capabilities without registering an intention.
+   * Another application that provides a public capability that matches the intent, however, should not receive it.
```

Comment:
What do you think of the following description?

`Tests that an application can issue intents to its public capabilities without registering an intention.
However, because not explicitly registered an intention, the intent should not be transported to other applications which provide a matching, public capability.`

Reply:
Done